Sources/SpeziValidation/ValidationState/ReceiveValidationModifier.swift

```diff
@@ -46,12 +36,8 @@ extension View {
     /// - Parameter state: The binding to the ``ValidationState``.
     /// - Returns: The modified view.
     public func receiveValidation(in state: ValidationState.Binding) -> some View {
-        let binding = IsolatedValidationBinding(state)
-
-        return onPreferenceChange(CapturedValidationStateKey.self) { entries in
-            Task { @Sendable @MainActor in
-                binding.state.wrappedValue = ValidationContext(entries: entries)
-            }
```

Comment:
I guess this preference closure "scheduling" broke some tests in modules using SpeziViews?

Reply:
yep, one of the AccountSignupView (i think?) tests expects that the Login button is disabled by default, as no username/password are entered yet, but the test fails because the snapshot of the view is taken before the Task updating the validation can run (thereby leaving the button in an enabled state)